02 Algorithm Reference/10 Rolling Window/01 Introduction.html

```diff
@@ -5,7 +5,7 @@
 <meta name="tag" content="indicators">
 <meta name="tag" content="rolling windows">
 <p>
-$[RollingWindow] is an array of data that allows for reverse list access semantics, where the object with index [0] refers to the <i>most recent item</i> in the window and index [Length-1] refers to the last item in the window, where Length is the number of elements in the window.
+<code>RollingWindow</code> is an array of data that allows for reverse list access semantics, where the object with index [0] refers to the <i>most recent item</i> in the window and index [Length-1] refers to the last item in the window, where Length is the number of elements in the window.
```

Comment:
`$[RollingWindow,T:QuantConnect.Indicators.RollingWindow]`

Reply:
I tried this `T:QuantConnect.Indicators.RollingWindow`, it doesn't work
https://www.quantconnect.com/services/inspector?type=T:QuantConnect.Indicators.RollingWindow

Reply:
`RollingWindow<T>` is a generic type, not sure if the inspector curerntly supports queries for this